src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -242,13 +234,6 @@ public void KeyboardInput_SquareRoot()
             Assert.AreEqual("10", page.GetCalculatorResultText());
         }
 
-        [TestMethod]
-        public void KeyboardInput_Cubed()
-        {
-            page.Header.SendKeys("3#");
-            Assert.AreEqual("27", page.GetCalculatorResultText());
-        }
```

Comment:
Is this test not valid any more or does it just need to be updated?

Reply:
Not valid, button no longer exists in standard mode